content/rest-api/index.textile

```diff
@@ -630,18 +634,56 @@ bc[sh]. curl -X PUT https://rest.ably.io/push/deviceRegistrations/01ARZ3NDEKTSV4
  -u "{{API_KEY}}"
  -H "Content-Type: application/json" \
  --data \
- '
- {
+'{
+  "id": "01ARZ3NDEKTSV4RRFFQ69G5FAV",
+  "platform": "ios",
+  "formFactor": "phone",
   "metadata": {
-    "myKey": "value"
+    "timezone": "PST"
   },
   "push": {
     "recipient": {
+      "transportType": "apns",
       "deviceToken": "740f4707bebcf74f9b7c25d48e3358945f6aa01da5ddb387462c7eaf61bb78ad"
     }
   }
-}
-'
+}'
+
+h5. Parameters
+
+None
+
+h5. Options
+
+- Content-Type := @application/json@, @application/x-msgpack@ or @application/x-www-form-urlencoded@
+- Accept := @application/json@ by default, or @application/x-msgpack@, @text/html@
+- Auth required := yes ("basic":#basic-authentication or "token":#token-authentication with @push-admin@ capability)
+
+A successful request returns the updated device details.
+
+An unsuccessful request returns an error.
+
+h6. PATCH rest.ably.io/push/deviceRegistrations/@<device id>@
+
+The body must have the same shape as when "registering the device":#post-device-registration, except only fields to be changed should be provided. Any fields provided replace existing values. Please bear in mind that fields that are object (Hash) types will replace existing values as opposed to be merged together. @metadata@ and @push.recipient@ are examples of these types. Currently only the following attributes are currently updateable and any attempt to modify other fields will result in the update failing:
```

Comment:
Hash is pretty Ruby-specific. I'd say something like "fields whose values are structured (JSON-like arrays or objects)".

s/updateable/updatable/, and there's a duplicate "currently" in "Currently only the following attributes are currently" (I'd really remove both; all this documentation is "currently" implicitly).